### PR TITLE
replace bucketId by bucketName

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**
add session id to catalog request

reformat code

### CODE REVIEW

**Review by jrochas — APPROVED** [review]

**Review by aminelouati — APPROVED** [review]

**Comment by Mykhailenko** [comment]
retest this please

File: src/main/java/org/ow2/proactive/catalogclient/service/CatalogObjectService.java [hunk 1]
```diff
@@ -118,19 +118,19 @@ public String getResolvedCatalogObject(String catalogUrl, long bucketId, String
...
      * @param name is the resource name
      * @param raw enables to choose for the resource metadata or the resource content
      * @return the resource content if raw is true otherwise return the resource metadata
      */
     @VisibleForTesting
-    String getURL(String catalogURL, long bucketId, String name, boolean raw) {
+    String getURL(String catalogURL, String bucketName, String name, boolean raw) {
 
         final String bucketsPath = "buckets/";
         final String resourcesPath = "/resources/";
```
Comment (sonarqube-activeeon) [inline src/main/java/org/ow2/proactive/catalogclient/service/CatalogObjectService.java]: MINOR Refactor your code to get this URI from a customizable parameter. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)

File: src/main/java/org/ow2/proactive/catalogclient/service/CatalogObjectService.java [hunk 2]
```diff
@@ -118,19 +118,19 @@ public String getResolvedCatalogObject(String catalogUrl, long bucketId, String
...
         final String rawPath = "/raw";
```
Comment (sonarqube-activeeon) [inline src/main/java/org/ow2/proactive/catalogclient/service/CatalogObjectService.java]: MINOR Refactor your code to get this URI from a customizable parameter. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 4 issues
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [CatalogObject.java#L87](https://github.com/ow2-proactive/catalog-client/blob/0123868f7fc8fc2e603ce1fe616e3529c9d65540/src/main/java/org/ow2/proactive/catalogclient/model/CatalogObject.java#L87): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [CatalogObject.java#L90](https://github.com/ow2-proactive/catalog-client/blob/0123868f7fc8fc2e603ce1fe616e3529c9d65540/src/main/java/org/ow2/proactive/catalogclient/model/CatalogObject.java#L90): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Mykhailenko** [comment]
retest this please

**Comment by Mykhailenko** [comment]
retest this please

File: src/main/java/org/ow2/proactive/catalogclient/service/CatalogObjectService.java (hunk 1 above)
Comment (sonarqube-activeeon) [inline src/main/java/org/ow2/proactive/catalogclient/service/CatalogObjectService.java]: MINOR Refactor your code to get this URI from a customizable parameter. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)

File: src/main/java/org/ow2/proactive/catalogclient/service/CatalogObjectService.java (hunk 2 above)
Comment (sonarqube-activeeon) [inline src/main/java/org/ow2/proactive/catalogclient/service/CatalogObjectService.java]: MINOR Refactor your code to get this URI from a customizable parameter. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 4 issues
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [CatalogObject.java#L87](https://github.com/ow2-proactive/catalog-client/blob/0123868f7fc8fc2e603ce1fe616e3529c9d65540/src/main/java/org/ow2/proactive/catalogclient/model/CatalogObject.java#L87): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [CatalogObject.java#L90](https://github.com/ow2-proactive/catalog-client/blob/0123868f7fc8fc2e603ce1fe616e3529c9d65540/src/main/java/org/ow2/proactive/catalogclient/model/CatalogObject.java#L90): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**